packs/feat-effects/effect-sheltering-pulse.json

```
@@ -0,0 +1,51 @@
+{
+    "_id": "m0hi0jpIF6tNJzzo",
+    "img": "icons/magic/defensive/barrier-shield-dome-blue-purple.webp",
+    "name": "Effect: Sheltering Pulse",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Sheltering Pulse]</p>\n<p>You and your allies gain a +1 status bonus to AC.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 3
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Lost Omens: Tian Xia Character Guide"
+        },
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "sheltering-pulse",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "sheltering-pulse"
+                ],
+                "selector": "ac",
+                "type": "status",
+                "value": 1
+            }
+        ],
```

```suggestion
        "rules": [
            {
                "key": "FlatModifier",
                "selector": "ac",
                "type": "status",
                "value": 1
            }
        ],
```
No need for the toggle here